src/codeu/chat/common/User.java

```diff
@@ -47,6 +51,9 @@ public User read(InputStream in) throws IOException {
 
     }
   };
+  
+  public HashMap<Uuid, Time> UserUpdateMap = new HashMap<Uuid, Time>();
```

Comment:
nit: camelcase for instance variables (should be userUpdateMap and convoUpdateMap)